lib/streamlit/hashing.py

```diff
@@ -493,6 +494,15 @@ def _to_bytes(self, obj, context):
         elif type_util.is_type(obj, "tensorflow.python.client.session.Session"):
             return self.to_bytes(id(obj))
 
+        elif type_util.is_type(obj, "builtins.PyCapsule"):
+            ctypes.pythonapi.PyCapsule_GetPointer.restype = ctypes.c_void_p
```

Comment:
Move the `ctypes` import over here. One thing we try hard with Streamlit is to reduce the library's load time, so imports should take place as close as possible to their usage.